### PR TITLE
Drag and select + editable proposals card from table

### PR DESCRIPTION
**Pull request by Devorein**
### WHY

[Card](https://app.charmverse.io/charmverse/proposals-updated-column-5785901663768147)
[Card](https://app.charmverse.io/charmverse/multi-select-and-edit-rows-in-proposal-table-21506491975060316)

### CODE REVIEW

File: components/common/BoardEditor/components/properties/ErrorWrapper.tsx [hunk 1]
```diff
@@ -1,10 +1,16 @@
-import { Box, Stack, Typography } from '@mui/material';
+import { Stack, Typography } from '@mui/material';
 
 export function ErrorWrapper({ error, children }: { error?: string; children: React.ReactNode }) {
   return (
-    <Stack flex={1}>
+    <Stack
+      flex={1}
+      sx={{
```
Comment (Devorein) [inline components/common/BoardEditor/components/properties/ErrorWrapper.tsx:7]: Without this the person property cell is only a single px tall and thus the select dropdown can't be triggered easily

File: components/common/BoardEditor/focalboard/src/components/table/tableRow.tsx [hunk 2]
```diff
@@ -224,8 +225,15 @@ function TableRow(props: Props) {
       }}
     >
       {!props.readOnly && (
-        <>
-          <div className='icons row-actions' onClick={handleClick} ref={drag}>
+        <Stack flexDirection='row' gap={1} alignItems='center'>
```
Comment (Devorein) [inline components/common/BoardEditor/focalboard/src/components/table/tableRow.tsx:228]: This enables dragging and selecting table rows from the top of the checkboxes

**Comment by motechFR** [comment]
Running into this error when navigating into proposals on staging, can you reproduce?

Screenshot 2023-12-29 at 13 23 42

File: components/common/BoardEditor/focalboard/src/components/propertyValueElement.tsx [hunk 3]
```diff
@@ -304,6 +306,26 @@ function PropertyValueElement(props: Props) {
         showEmptyPlaceholder={showEmptyPlaceholder}
       />
     );
+  } else if (propertyTemplate.type === 'proposalAuthor') {
```
Comment (motechFR) [inline components/common/BoardEditor/focalboard/src/components/propertyValueElement.tsx:309]: One suggestion, we shouldn't remove the original author, just add some new ones

Comment (Devorein) [reply components/common/BoardEditor/focalboard/src/components/propertyValueElement.tsx:309]: Currently we don't let the user make the authors empty, I will talk with Chris and ask his opinion regarding this.

**Pull request merged**